packages/flutter/lib/src/widgets/scrollbar.dart

```diff
@@ -90,6 +90,7 @@ class ScrollbarPainter extends ChangeNotifier implements CustomPainter {
     double minLength = _kMinThumbExtent,
     double? minOverscrollLength,
     ScrollbarOrientation? scrollbarOrientation,
+    bool ignorePointer = false,
```

Comment:
Rather than enumerate more scrollbar parameters, we could probably do this very simply in just the RawScrollbar.

Reply:
We should assert this is not null. I know in null-safety and that is unnecessary, but we still have these assertions in place for those who have not migrated yet.